### PR TITLE
Feature/end2end

### PR DESCRIPTION
**Pull request by hellais**

### CODE REVIEW

File: backend/globaleaks/handlers/wbtip.py [hunk 1]
```diff
@@ -237,28 +239,11 @@ def localize_and_append_receiver(receiver, receiver_desc):
...
-                "name": rtip.receiver.name,
-                "id": rtip.receiver.id,
-                "pgp_key_status": rtip.receiver.pgp_key_status,
-                "access_counter": rtip.access_counter,
-                "message_counter": message_counter,
-                "creation_date": datetime_to_ISO8601(datetime_now()),
-            }
-
-            localize_and_append_receiver(rtip.receiver, receiver_desc)
-
```
Comment (hellais) [inline backend/globaleaks/handlers/wbtip.py]: This should not be missing.

File: backend/globaleaks/handlers/wbtip.py [hunk 2]
```diff
@@ -237,28 +239,11 @@ def localize_and_append_receiver(receiver, receiver_desc):
                 "access_counter": 0,
                 "message_counter": 0,
                 "creation_date": datetime_to_ISO8601(datetime_now()),
+                "pgp_glkey_pub": receiver.pgp_glkey_pub,
```
Comment (hellais) [inline backend/globaleaks/handlers/wbtip.py]: XXX

File: backend/globaleaks/rest/requests.py [hunk 3]
```diff
@@ -71,9 +73,17 @@
...
+
+    #EEE
+    #'gpg_key_armor': unicode,
+    #'pgp_key_armor_priv': unicode,
+
+    'pgp_glkey_pub': unicode,
+    'pgp_glkey_priv': unicode,
     "tip_notification": bool,
     "ping_notification": bool,
-    "language": unicode,
```
Comment (hellais) [inline backend/globaleaks/rest/requests.py]: ?? Why is this commented out?

**Comment by hellais** [comment]
I get this error when running globaleaks:

```
2015-03-31 18:57:07+0200 '[!] Failed Context initialization no such table: receiver'
2015-03-31 18:57:07+0200 "[!] Uncaught exception <class 'sqlite3.OperationalError'> no such table: receiver <traceback object at 0x10c238248>"
2015-03-31 18:57:07+0200 [ ] no such table: receiver
2015-03-31 18:57:07+0200 '[!] sqlite3.OperationalError None'
2015-03-31 18:57:07+0200 '[!] Traceback (most recent call last):\\n\\n  File "/Users/x/.virtualenvs/globaleaks/lib/python2.7/site-packages/twisted/internet/defer.py", line 1097, in _inlineCallbacks\\n    result = result.throwExceptionIntoGenerator(g)\\n\\n  File "/Users/x/.virtualenvs/globaleaks/lib/python2.7/site-packages/twisted/python/failure.py", line 389, in throwExceptionIntoGenerator\\n    return g.throw(self.type, self.value, self.tb)\\n\\n  File "/Users/x/code/GL/GlobaLeaks/backend/globaleaks/handlers/wizard.py", line 73, in post\\n    yield wizard(request, self.request.language)\\n\\n  File "/Users/x/.virtualenvs/globaleaks/lib/python2.7/site-packages/twisted/python/threadpool.py", line 191, in _worker\\n    result = context.call(ctx, function, *args, **kwargs)\\n\\n  File "/Users/x/.virtualenvs/globaleaks/lib/python2.7/site-packages/twisted/python/context.py", line 118, in callWithContext\\n    return self.currentContext().callWithContext(ctx, func, *args, **kw)\\n\\n  File "/Users/x/.virtualenvs/globaleaks/lib/python2.7/site-packages/twisted/python/context.py", line 81, in callWithContext\\n    return func(*args,**kw)\\n\\n  File "/Users/x/code/GL/GlobaLeaks/backend/globaleaks/settings.py", line 798, in _wrap\\n    result = function(self.store, *args, **kwargs)\\n\\n  File "/Users/x/code/GL/GlobaLeaks/backend/globaleaks/handlers/wizard.py", line 44, in wizard\\n    raise excep\\n\\nOperationalError: no such table: receiver\\n'
2015-03-31 18:57:07+0200 '[!] Exception mail! [2]'
2015-03-31 18:57:07+0200 Starting factory <twisted.mail.smtp.ESMTPSenderFactory instance at 0x10bcc68c0>
2015-03-31 18:57:07+0200 [http] 500 POST /admin/wizard (127.0.0.1) 351.86ms
2015-03-31 18:57:08+0200 Stopping factory <twisted.mail.smtp.ESMTPSenderFactory instance at 0x10bcc68c0>

```

**Comment by evilaliv3** [comment]
closing this pull; all the relevant code has been ported or archived.

**Pull request closed without merging**